lib/utilities/constant.dart

```diff
@@ -23,5 +16,15 @@ const addOneToPokemonIndex = 1;
 const basedImageUrl = 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/home/[index].png';
 const indexKey = '[index]';
 
+// Spacing
+const spacingXLarge = 50.0;
```

Comment:
seems not needed, just directly use this where needed 🙂 

you can already get rid of these spacing constants up to `spacingInfinity`